### ISSUE TITLE
"hello" at the top of the window helper

### ISSUE THREAD

**Issue by Suisse00** [bombledmonk/advancedsearch#10]
When I click one of your button (checkboxes, helpers, ...) the first line is always "hello". Look like a POC hard-coded!

**Comment by bombledmonk** [comment]
Yeah, I've been staring at that for years.... It's never been convenient to fix when I've noticed.  I'll fix it because someone asked.  Thanks for your feedback.

**Comment by Suisse00** [comment]
Well I also did create tickets to maybe try to fix some when I will have time.

I still don't know why digikey didnt yet bother to make something clear for filter... I'm looking for parts no that often and I always hate that. RIP anybody using it on a daily base.

So thanks to have created this in the first place ;)

**Comment by bombledmonk** [comment]
Good luck with the code base.... This was a learning project started in 2011ish when I knew literally zero about javascript/webdev.  It's pretty hacked together and poorly documented( past self was not kind to future self).  I've rewrote several pieces over time, but it's still pretty tough for me to follow, let alone someone else.
